malos/driver.proto

```diff
@@ -329,6 +329,9 @@ message LircParams {
 
   // IR command
   string command = 2;
+
+  // Set config
```

Comment:
This is not "set config". Pleas add a more descriptive comment here.

Like:

// IR configuration used for X. It should be in the format XXXX.
// For instance: ZZZZ.